### ISSUE TITLE
ORM Model register failed

### ISSUE THREAD

**Issue by wadehehe** [asif-mahmud/eve_runner#1]
Base._decl_class_registry can not find a key '__tablename__'

What reason cause this?

I have already inherit BaseModel

**Comment by asif-mahmud** [comment]
did you declare `__tablename__` in your model class ? if not that should be the reason.

**Comment by asif-mahmud** [comment]
BTW, you should utilize `db.ModelBase` for database models, it handles the tablename internally. check the source in `instance/db.py` -> `ModelBase`

**Comment by wadehehe** [comment]
Yes  I declare it. I think i have resolved it , ModelBase should inherit  Sqlalchemy Model not Object.

**Comment by asif-mahmud** [comment]
sorry, i had to recheck my own reply, its in the git's readme file - you have to use `instance.db.Base` as your model's base. that will do.

**Comment by wadehehe** [comment]
Thanks ,I also try with instance.db.Base ,but it also not working.
 I think _decl_class_registry is sqlalchemy attribute, so we must use it as ModelBase's base.

**Comment by asif-mahmud** [comment]
no no, you have to use `db.Base` not the `ModelBase`.

Best wishes -
Asif Mahmud Shimon
